### PR TITLE
Fixes in website

### PR DESCRIPTION
**Pull request by jonahlikescookies**
### Description

Addresses some requested changes in #16, the default profile picture scaling and separate scroll boxes for packs and comments. Edit: I forgot about this, I also shrank the modal. There are a couple of things I would like to work on before merging, including:
- Avoiding the usage of dvw and dvh
- Fixing the body having a scroll box

### Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by MesterMan03** [comment]
The new changes created a giant void in the bottom of the viewport which cuts off content, is this intentional?
image

**Comment by jonahlikescookies** [comment]
Yes, it is intentional, it is there to avoid another scroll box being created for the body. If you want, you can increase the height of the sections in order to fill the void. However, this may not fix it for all devices, as the heading size is fixed.

**Comment by MesterMan03** [comment]
I tried addressing some of the issues, now both comments and packs take up the entire space and they're still independently scrollable.

One issue is that if <main> doesn't take up only the **rest** of the available height, then a little content gets chopped off when you scroll to the bottom. I fixed this by setting the height to 90% (for header 10%), but this has some minor defects in special conditions.

Basically, this solution becomes perfect if <main> takes up the remaining height of <body>. Worst case scenario, we'll have to use some super ugly JavaScript code to manually calculate the width if it's not possible by CSS.

**Comment by jonahlikescookies** [comment]
I would try calculating the width with the CSS [`calc()` method](https://developer.mozilla.org/en-US/docs/Web/CSS/calc).

**Comment by jonahlikescookies** [comment]
@MesterMan03 I currently don’t have access to a computer, so I am unable to thoroughly test my changes. Can you make sure that the scaling works fine for different viewport sizes?

**Review by MesterMan03 — APPROVED** [review]
It looks great to me. If there's anything else you'd like, let me know, otherwise LGTM.

**Comment by jonahlikescookies** [comment]
I think we can add CSS for the pack selection for comments before merging, as it definitely looks different across browsers.

**Comment by jonahlikescookies** [comment]
If you don’t like how I styled the `<select>` elements, then feel free to make any changes. Otherwise, feel free to merge.

**Comment by MesterMan03** [comment]
Alright cool, we're slowly getting closer to a public release. The biggest challenge will probably be the mobile layout, but we might get away with it since I presume most people will use computers to download Minecraft resource packs.
